packages/airnode-examples/integrations/README.md

```diff
@@ -29,6 +29,7 @@ When adding an integration you need to do a few things:
    the data received by the Airnode
 3. Create all the necessary files (see the section above) for your integration. Some of the files are explained more in
    depth below
+4. Run `yarn build` from the top-level `airnode` directory to build the contract artifacts.
```

Comment:
:+1:

Reply:
Actually, this is not enough - also the users need to run this to compile the contracts.

We should add this as a step to the "main" README